__mocks__/@actions/core.ts

```diff
@@ -0,0 +1,9 @@
+const self = {
+    error: jest.fn(),
+    info: jest.fn(),
+    debug: jest.fn(),
+    warn: jest.fn(),
+    getInput: jest.fn(),
+    addPath: jest.fn(),
+}
+export default self;
```

Comment:
Why did you need to specify those? 
Didn't jest automocked these for you?

Reply:
nope. @actions/core does some type voodoo, mocking it with jest without this file causes errors such as "undefined does not have `addPath` property". i didn't research why that is.